alpine/resin

```diff
@@ -0,0 +1,66 @@
+#!/sbin/openrc-run
```

Comment:
This service is a bit complex since I need to manually clean up all the processes created by the service. It depends on how users write the CMD command in Dockerfile (shell form or exec form)  that creates many child processes while OpenRC only handles parent process (process in resinapp.pid).
I'm thinking about using process supervision like S6 https://github.com/OpenRC/openrc/blob/master/s6-guide.md to simplify our service